packages/core/src/MultiButton/MultiButton.styles.tsx

```diff
@@ -47,6 +47,9 @@ export const { staticClasses, useClasses } = createClasses("HvMultiButton", {
           borderRightColor: theme.colors.border,
         },
       },
+      "&$firstButton$lastButton": {
+        borderRadius: theme.radii.base,
+      },
```

Comment:
this is doing nothing. DS5 doesn't have this bug.
also, we should use `"inherit"` for these cases. this is making it harder to override